spec/features/trainees/submit_for_trn_spec.rb

```diff
@@ -12,9 +12,25 @@
 
   describe "submission" do
     context "when all sections are completed" do
-      before do
-        given_a_trainee_exists(:with_programme_details, degrees: [create(:degree, :uk_degree_with_details)])
-        stub_progress_service(completed: true)
+      let(:trainee) do
+        create(
```

Comment:
Wondering if we should create a factory for this to quickly spin up a "full and valid trainee"

Reply:
Good idea 👍